app/assets/javascripts/client/inventories/add_inventory_users_link.html

```diff
@@ -0,0 +1,3 @@
+<div>
+  <div ng-click="link.open()">Add Users</div>
```

Comment:
where can I see the style of the links to be used here? we're going to use there in the inventory details page, it should match. do you have it?